src/rules/no_const_assign.rs

```diff
@@ -210,25 +226,70 @@ mod tests {
 
   #[test]
   fn no_const_assign_invalid() {
-    assert_lint_err::<NoConstAssign>("const x = 0; x = 1;", 13);
-    assert_lint_err::<NoConstAssign>("const {a: x} = {a: 0}; x = 1;", 23);
-    assert_lint_err::<NoConstAssign>("const x = 0; ({x} = {x: 1});", 15);
-    assert_lint_err::<NoConstAssign>("const x = 0; ({a: x = 1} = {});", 14);
-    assert_lint_err::<NoConstAssign>("const x = 0; x += 1;", 13);
-    assert_lint_err::<NoConstAssign>("const x = 0; ++x;", 13);
-    assert_lint_err::<NoConstAssign>(
-      "const x = 0; function foo() { x = x + 1; }",
-      30,
-    );
-    assert_lint_err::<NoConstAssign>(
-      "const x = 0; function foo(a) { x = a; }",
-      31,
-    );
-    assert_lint_err::<NoConstAssign>("for (const i = 0; i < 10; ++i) {}", 26);
-    assert_lint_err::<NoConstAssign>(
-      "const x = 0; while (true) { x = x + 1; }",
-      28,
-    );
+    assert_lint_err! {
+      NoConstAssign,
+      r#"const x = 0; x = 1;"#: [
+      {
+        col: 13,
+        message: NoConstantAssignMessage::Unexpected,
+        hint: NoConstantAssignHint::Remove,
+      }],
+      r#"const {a: x} = {a: 0}; x = 1;"#: [
+      {
+        col:  23,
```

Comment:
I'd be happy if you could remove extra whitespaces after `col:` for all the test cases 👍

Reply:
Thanks! Looks like there are still undone cases. So can you please do the same for the following tests:

- `triple_slash_reference_invalid`
- `single_var_declarator_invalid`
- `no_throw_literal_invalid`
- `no_sparse_arrays_invalid`

Reply:
Done in dd6173c